test/types/3d-secure.ts

```diff
@@ -0,0 +1,14 @@
+export default function threeDSecure() {
+    const risk = window.recurly.Risk();
+    const threeDSecure = risk.ThreeDSecure({
+        actionTokenId: 'token',
+    });
+
+    threeDSecure.on('token', function() {});
+
+    threeDSecure.on('error', function() {});
+
+    let el: HTMLElement;
+
+    threeDSecure.attach(el);
+}
```

Comment:
Looks like all of the ts files are using a spacing width of 4 when the rest of the lib uses 2